src/transaction/locator_sr.c

```diff
@@ -13798,10 +13798,15 @@ locator_multi_insert_force (THREAD_ENTRY * thread_p, HFID * hfid, OID * class_oi
 						     func_preds, force_in_place, &home_hint_p, has_BU_lock, dont_check_fk, true);
 		  if (error_code != NO_ERROR)
 		    {
-		      pgbuf_ordered_unfix_and_init (thread_p, home_hint_p.pgptr, &home_hint_p);
+		      //pgbuf_ordered_unfix_and_init (thread_p, home_hint_p.pgptr, &home_hint_p);
+                      ASSERT_ERROR ();
+
+                      if (pgbuf_is_page_fixed_by_thread (thread_p, &new_page_vpid))
```

Comment:
better to unfix both home_hint_p.pgptr and scan_cache->page_watcher.pgptr

Reply:
also better not to use `pgbuf_is_page_fixed_by_thread` that is a debugging function:
```c
if (home_hint_p.pgptr != NULL)
  pgbuf_ordered_unfix_and_init (..., home_hint_p);
```